subsys/bluetooth/host/iso.c

```diff
@@ -1140,7 +1153,9 @@ static void cleanup_big(struct bt_iso_big *big)
 
 	big->bis = NULL;
 	big->num_bis = 0;
-	atomic_clear(&big->initialized);
+	atomic_clear_bit(big->flags, BT_BIG_SYNCING);
+	atomic_clear_bit(big->flags, BT_BIG_PENDING);
+	atomic_clear_bit(big->flags, BT_BIG_INITIALIZED);
```

Comment:
```suggestion
	atomic_clear(&big->flags);
```

Reply:
It's not the same type:
```
warning: passing argument 1 of 'atomic_clear' from incompatible pointer type [-Wincompatible-pointer-types]
 1156 |  atomic_clear(&big->flags);
 ```
 
 Could update to a memset of the struct?

Reply:
Might just be `atomic_clear(big->flags);`. We could memset the entire struct as well as that's what we are basically doing anyhow.